packages/permissions/src/components/branch-on-permissions/branch-on-permissions.spec.tsx

```diff
@@ -55,15 +55,13 @@ const renderWithPermissions = (demandedPermissions: string[]) => {
         suspension: {
           isActive: true,
         },
-        allAppliedPermissions: [{ name: 'canViewProducts', value: true }],
-        allAppliedActionRights: [
-          {
-            group: 'products',
-            name: 'canEditPrices',
-            value: false,
+        permissions: { canViewProducts: true },
```

Comment:
Why did you change those? 🤔

Reply:
They need to stay as they were

Reply:
I thought those were deprecated. I changed the tests to reflect the new params (permissions, dataFences, actionRights). also, typescript was complaining about them inside the test...

Reply:
I'm not sure what you mean with "deprecated" but this is what we get from the graphql query

Reply:
```
// For backwards compatibility we need to denormalize the given `permissions` option
// (which is now deprecated) to `allAppliedPermissions`, in order to pass the value
// to the `project` prop in the application context provider.
```

I got this backwards 😓

Reply:
reverted 🙏